pkg/controllers/networking/pod_controller.go

```diff
@@ -123,6 +124,25 @@ func (r *PodReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result
 		return ctrl.Result{}, nil
 	}
 
+	// It is not allowed that specified mac address is in conflict with existing IPInstances.
```

Comment:
can you check this in `createIP` of `IPAMStore`?

Reply:
Okay! It makes conflicts checked in a "global" manner.